content/concepts/decentralized-exchange/automated-market-makers.md

```diff
@@ -0,0 +1,88 @@
+---
+html: automated-market-makers.html
+parent: decentralized-exchange.html
+blurb: Automated Market Makers (AMMs) provide liquidity between asset pairs, complemeting the order books in the decentralized exchange while providing passive income for their liquidity providers.
+status: not_enabled
+labels:
+  - XRP
+  - Decentralized Exchange
+  - AMM
+---
+# Automated Market Makers
+
+An Automated Market Maker (AMM) is a type of smart contract, built into the XRP Ledger, that provides liquidity in the decentralized exchange. An AMM holds a pool of two assets, which can be XRP or [tokens](tokens.html), and allows users to swap between them at an exchange rate set by a formula. [Offers](offers.html) and [Cross-Currency Payments](cross-currency-payments.html) automatically use a combination of Offers and AMMs to achieve cost-effective exchanges between assets.
+
+{% include '_snippets/amm-disclaimer.md' %}
+
+For any given pair of assets, there can be up to one AMM in the ledger. Anyone can create an AMM for an asset pair if it doesn't exist yet, or deposit to an existing AMM's pools. Those who deposit assets into an AMM are called _liquidity providers_ (LPs) and receive "LP Tokens" from the AMM in proportion to the amounts they deposit. LP Tokens are like other fungible tokens in the XRP Ledger, except that they also give special privileges for the AMM that issues them:
+
+- Liquidity providers can redeem their LP Tokens to withdraw a corresponding share of the assets in the AMM's pool, including fees collected.
+- Liquidity providers can vote to change the AMM's fee settings. The votes are weighted based on how many LP Tokens the voters hold.
+- Liquidity providers can bid some of their LP Tokens to receive a temporary discount on the AMM's trading fees.
+
+When the flow of funds between the two assets in a pool is relatively active and balanced, the fees provide a source of passive income for liquidity providers. However, when the relative price between the assets shifts, the liquidity providers can take a loss on the [currency risk](https://www.investopedia.com/terms/c/currencyrisk.asp).
+
+## How the AMM Works
+
+[Kris Machowski's Introduction to Automated Market Makers](https://www.machow.ski/posts/an_introduction_to_automated_market_makers/) is a good in-depth explanation on the economics and principles of AMMs in general. The XRP Ledger implements a _constant product_ AMM, which sets the exchange rate between the two assets in its pool so that multiplying the amounts of the two pools together always comes out to the same value. For example, if the AMM holds 5 ETH and 5 USD, the "constant product" value is 5×5=25. To buy 1 ETH, you'd have to spend 1.25 USD, because after removing 1 ETH from the pool and adding 1.25 USD, the constant product value is 4×6.25=25. The AMM also charges a configurable trading fee, which can be as low as 0 or as high as 1%, so you would have to pay a little more for that as well.
```

Comment:
```suggestion
The XRP Ledger implements a _constant product_ AMM, which determines the exchange rate by the ratio of assets in the pool. As the quantity of one asset goes down, the exchange rate to the other asset goes up. To see a detailed breakdown of the equation used, see: [Kris Machowski's Introduction to Automated Market Makers](https://www.machow.ski/posts/an_introduction_to_automated_market_makers/#constant-product-amms). The AMM also charges a trading fee between 0 and 1%, which is determined and redeemed by LPs.
```
Since it's a conceptual topic I'd try and keep it high-level still. The equation portion is meaty enough that they _should_ go look up how it works on the external site, but they only need to know the gist of it to grasp how the AMM works here. May want to get a second opinion on my one-liner explanation.